pkg/server/nfs/nfs_server.go

```diff
@@ -133,3 +145,25 @@ func setRlimitNOFILE(logger logrus.FieldLogger) error {
 	logger.Infof("ending RLIMIT_NOFILE rlimit.Cur %d, rlimit.Max %d", rlimit.Cur, rlimit.Max)
 	return nil
 }
+
+func SetConfigParams(config string) string {
```

Comment:
Add SetConfigParams to determine the log path according to process is LH manager or not.